app.js

```diff
@@ -0,0 +1,21 @@
+const express = require("express");
+const app = express();
+app.use(express.json());
```

Comment:
this can be removed as we do need that piece of middleware just yet

Reply:
Thanks! I've removed it :)